examples/lab62/server.py

```diff
@@ -19,9 +19,16 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-import SocketServer
+from __future__ import absolute_import
+from __future__ import print_function
+
 import re
 
+try:
+    import SocketServer as socketserver
+except ImportError:
+    import socketserver
+
```

Comment:
Updated the import. 

> BTW: this example should be changed to use the mllp module. It is legacy code written before we implemented MLLP

I don't really know enough about MLLP to rework the example, since we only use the parser. Reworking the example seems unrelated to updating to Python 3. Is this a requirement to get this merged?